libproject/workbench.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "utils/common"
+
+module Workbench
+  WORKBENCH_ROOT = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+
+  def check_submodules()
+    # `git clone` includes submodule folders but nothing else.
+    unless File.exists? File.join(WORKBENCH_ROOT, "libproject", "utils", "README.md")
+      unless system(*%W{git submodule update --init})
+        STDERR.puts "`git submodule update` failed."
```

Comment:
Worth noting: `common.error` prints this in bold

Reply:
Switched! Thanks.